docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -298,6 +298,120 @@ Followings are the available elements:
+        - EQ: Check if CV == SV
+        - NEQ: Check if CV != SV
+        - GT: Check if CV > SV
+        - GE: Check if CV >= SV
+        - LT: Check if CV < SV
+        - LE: Check if CV <= SV
+        - RANGE_INCLUSIVE: Check if SV1 <= CV and CV <= SV2
+        - RANGE_EXCLUSIVE: Check if SV1 < CV and CV < SV2
+        - NOT_IN_RANGE_INCLUSIVE: Check if CV < SV1 or SV2 < CV
+        - NOT_IN_RANGE_EXCLUSIVE: Check if CV <= SV1 or SV2 <= CV
+      - `then`: Specifies the action, if the result is TRUE.
+        - PASSTHROUGH: Does not let you make changes to the buffers. Buffers are pushed straight through.
+        - SKIP: Does not let you generate the output frame (frame skip).
+        - TENSORPICK: Lets you choose the nth tensor among the input tensors.
+          ```
+          [ tensor 0 ]
+          [ tensor 1 ]  ->   tensor if    ->    [ tensor 0 ]
+          [ tensor 2 ]    (tensorpick 0,2)      [ tensor 2 ]
+          input tensors                         output tensors
+          ```
+      - `then-option`: Specifies the option for TRUE action.
+        - nth tensor: used for TENSORPICK option, for example, `then-option`=0,2 means tensor 0 and tensor 2 are selected as output tensors among the input tensors.
+      - `else`: Specifies the action, if the result is FALSE.
+        - PASSTHROUGH: Does not let you make changes to the buffers. Buffers are pushed straight through.
+        - SKIP: Does not let you generate an output frame (frame skip).
+        - TENSORPICK: Lets you choose the nth tensor among the input tensors.
+      - `else-option`: Specifies the option for FALSE action.
+        - nth tensor: used for TENSORPICK option, for example, `else-option`=0,2 means tensor 0 and tensor 2 are selected as output tensors among the input tensors.
+
+    - **Example launch line with simple if-condition**
+
+      If a specific value ([3][4][2][5] value of the first tensor) is between a given range [10,100], input buffers are pushed straight through. If the value is not in given range, tensor 0 and tensor 2 are selected as output tensors. It can be expressed as:
+
+      ```
+      gst-launch ... (tensors 0,1,2 stream) !
+          tensor_if name=tif \
+                      compared-value=A_VALUE compared-value-option=3:4:2:5,0 \
+                      supplied-value=10,100 \
+                      operator=RANGE_INCLUSIVE \
+                      then=PASSTHROUGH \
+                      else=TENSORPICK \
+                      else-option=0,2 \
+          ! tif.src_0 ! (tensors 0,1,2 for TRUE action) ...
+          ! tif.src_1 ! (tensors 0,2 for FALSE action) ...
+      ```
+
+    - **Custom if-condition**
+
+      If the if-condition is complex and cannot be expressed with tensor_if expressions, you can define a custom if condition, and create a pipeline as follows:
+
+      - **Define and register if_custom**
```

Comment:
Confusion?
Is it if_custom or if-custom? 
      - **Define and register if_custom**

Reply:
Thanks.
I changed to ` **Define and register custom if-custom**`